src/roswire/common/msg.py

```diff
@@ -55,10 +50,42 @@ class Constant:
         The value of this constant.
     """
 
+    R_STRING_CONSTANT = re.compile("^\s*string\s+(\w+)\s*=\s*(.+)\s*$")
+    R_OTHER_CONSTANT = re.compile("^\s*(\w+)\s+(\w+)\s*=\s*([^\s]+).*$")
+
     typ: str
     name: str
     value: Union[str, int, float]
 
+    @classmethod
+    def from_string(cls, package: str, line: str) -> "Optional[Constant]":
+        """
+
```

Comment:
Missing one-line method description here and below.

Reply:
Done.